bridge/test/contract-mocks/bridgePool/ERC20Mock.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT-open-group
+pragma solidity ^0.8.11;
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ERC20Mock is ERC20 {
```

Comment:
Is it possible to use one of the existing ERC20 presets like [ERC20PresetMinterPauser](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/presets/ERC20PresetMinterPauser.sol) instead of creating our own Mock?